app/public/utils/dataProcessing/dataProcessingUtils.js

```diff
@@ -41,3 +41,32 @@ export const rowDisplayStyle = (isMarked, shouldHideMarkedRecords) => isMarked
         ? '.none'
         : '.row-selected'
     : '.row-not-selected';
+
+/**
+ * Converts bytes into human readable file size string
+ * @param {number} fileSizeInBytes in bytes
+ * @returns {string} human readable file size
+ */
+export const getReadableFileSizeString = (fileSizeInBytes) => {
+    const byteUnits = ['kB', 'MB', 'GB', 'TB', 'PB', 'EB', 'ZB', 'YB'];
+
+    const result = byteUnits.reduce((acc, _current) => acc.fileSize > 1024
+        ? { index: acc.index + 1, fileSize: acc.fileSize / 1024 }
+        : acc, { index: 0, fileSize: fileSizeInBytes / 1024 });
+
+    return `${Math.max(result.fileSize, 0.1).toFixed(1)} ${byteUnits[result.index]}`;
+};
+
+export const pageTitle = (page, pageNames) => {
+    switch (page) {
+        case pageNames.periods: return 'Periods';
+        case pageNames.runsPerPeriod: return 'Runs per period';
+        case pageNames.runsPerDataPass: return 'Runs per data pass';
+        case pageNames.dataPasses: return 'Data passes per period';
+        case pageNames.mc: return 'Monte Carlo';
+        case pageNames.flags: return 'Quality flags';
+        case pageNames.anchoragePerDatapass: return 'Anchorage per data pass';
+        case pageNames.anchoredPerMC: return 'Anchored per MC';
+        default: return page;
+    }
+};
```

Comment:
`pageTitle` doesn't belong to context of 'data processing utils'. I think it should be moved to other file

Reply:
I've made it a separate component.